### ISSUE TITLE
Breakage: nmap commands do not execute

### ISSUE THREAD

**Issue by jsrozner** [esm7/obsidian-vimrc-support#248]
**Please Review Before Posting!**

- [ x] I checked that the bug *does not happen* in the [CodeMirror Vim demo](https://codemirror-vim--util.repl.co/). If it does, please report it [there](https://github.com/replit/codemirror-vim) and not here.
- [x ] I'm reasonably sure that this bug is indeed about the Vimrc file support and not a general Vim in Obsidian issue. If it's a general Vim issue, report it [here](https://forum.obsidian.md/c/bug-reports/7).

**Describe the bug:**
Any command executed from vimrc with nmap not working, e.g.,
- `nmap zo :togglefold`
- `nmap zR :unfoldall`

```
exmap upskip jsfile .vim_js_helpers.js {moveUpSkipCollapsed(false)}
nmap j :upskip
```

**To Reproduce:**
- Use a command defined in vimrc
- for example, any of the above commands in describe the bug
- Observe that obsidian (in the bottom vim box) shows, e.g. `:goUp` or `:togglefold` (i.e. the command does not execute)
- Enter must be pressed to actually finish execution of the vim command (then it executes as expected)

** Comments **
It's my guess that a recent obsidian change broke this. But I am not sure if it's on them or on vimrc? The native vim plugin still does work - I just disabled my vimrc for now :(

They may have made a change that requires vimrc to change how command gets executed in call to the vim engine?

**Environment (please complete the following information):**

 - OS: mac 14.4.1
 - Vimrc plugin version: 0.9.0
 - Obsidian: 1.7.5

**Additional context:**

**Comment by grugq** [comment]
I'm experiencing the same thing. 

I found that adding pressing enter after the command will make it execute. For example in normal mode using `zo` will do nothing, but `zo<CR>` (i.e. pressing Enter) will work as expected. 

I tried modifying my `vimrc` to add a <CR> at the end of commands but that did not work. 

There is an ex command buffer for some reason, and when I type a command it will expand to the full command that obsidian should execute e.g. using `zo` I get:

```
: togglefold
```

Pressing Enter will execute the command.

Hopefully that helps figure out the breaking problem.
